### PR TITLE
Added qty on hand

### PR DESCRIPTION
**Pull request by Wanl23**

### CODE REVIEW

File: Sandbox.ShoppingCart/Controllers/ProductController.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public ActionResult Overview()
         {
             var model = _productRepository.GetProducts();
 
-            return View (model);
+            return View ("Overview",model);
```
Comment (RusGraf) [inline Sandbox.ShoppingCart/Controllers/ProductController.cs]: Add a space after ","

File: Sandbox.ShoppingCart/Models/Product.cs [hunk 2]
```diff
@@ -11,5 +11,6 @@ public class Product
         public string Name { get; set; }
         public string Description { get; set; }
         public double Price { get; set; }
+        public double Qty { get; set; }
```
Comment (RusGraf) [inline Sandbox.ShoppingCart/Models/Product.cs]: Please change Qty to QtyOnHand or QtyAvailable. As Qty will have another will have a different meaning in the future cart.

File: Sandbox.ShoppingCart/Views/Product/Overview.cshtml [hunk 3]
```diff
@@ -15,12 +15,15 @@
                 <div class="caption">
                     <h4 class="group inner list-group-item-heading">@product.Name</h4>
                     <p class="group inner list-group-item-text">@product.Description</p>
+                    
                     <div class="row">
                         <div class="col-xs-12 col-md-6">
                             <p class="lead">
                                 $@product.Price</p>
+                            <p> Quantity on hand: </p>
```
Comment (RusGraf) [inline Sandbox.ShoppingCart/Views/Product/Overview.cshtml]: Насчет стиля, попробуй вот так: `@foreach (var product in Model)
        {
            <div class="item  col-xs-4 col-lg-4">
                <div class="thumbnail">
                    
                    <div class="caption">
                        <h4 class="group inner list-group-item-heading">@product.Name</h4>
                        <p class="group inner list-group-item-text">@product.Description</p>

                        <div class="row">
                            <div class="col-xs-12 col-md-6">
                                <p class="lead">
                                    Price: $@product.Price
                                </p>                                
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-xs-12 col-md-6">
                                <p class="lead">
                                    Qty available: 123
                                </p>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-xs-12 col-md-6">
                                <a class="btn btn-success" href="">Add to cart</a>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        }`

Comment (RusGraf) [inline Sandbox.ShoppingCart/Views/Product/Overview.cshtml]: Насчет стиля, попробуй вот так: 

`
@foreach (var product in Model)
        {
            <div class="item  col-xs-4 col-lg-4">
                <div class="thumbnail">
                    
                    <div class="caption">
                        <h4 class="group inner list-group-item-heading">@product.Name</h4>
                        <p class="group inner list-group-item-text">@product.Description</p>
                        <div class="row">
                            <div class="col-xs-12 col-md-6">
                                <p class="lead">
                                    Price: $@product.Price
                                </p>                                
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-xs-12 col-md-6">
                                <p class="lead">
                                    Qty available: 123
                                </p>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-xs-12 col-md-6">
                                <a class="btn btn-success" href="">Add to cart</a>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        }
`

File: Sandbox.ShoppingCart/Controllers/ProductController.cs (hunk 1 above)
Comment (Wanl23) [reply Sandbox.ShoppingCart/Controllers/ProductController.cs]: Done

File: Sandbox.ShoppingCart/Models/Product.cs (hunk 2 above)
Comment (Wanl23) [reply Sandbox.ShoppingCart/Models/Product.cs]: I changed to "QtyAvailable"

File: Sandbox.ShoppingCart/Views/Product/Overview.cshtml (hunk 3 above)
Comment (Wanl23) [reply Sandbox.ShoppingCart/Views/Product/Overview.cshtml]: I just copy all of this and put enters

Comment (Wanl23) [reply Sandbox.ShoppingCart/Views/Product/Overview.cshtml]: Why did u enter label as "Qty available: 123" and del @product.qty?

Comment (RusGraf) [reply Sandbox.ShoppingCart/Views/Product/Overview.cshtml]: Ohh my bad. Qty Available better suited to the locking style, removing @product.qty was a mistake.  I was playing with different numbers.

**Pull request merged**